internal/app/cli.go

```diff
@@ -48,6 +48,44 @@ func CreateClusterCommand(p PolicyAutomation) *cli.Command {
 		Name:  "cluster",
 		Usage: "Manage policies against GKE clusters",
 		Subcommands: []*cli.Command{
+			{
+				Name:  "json-data",
+				Usage: "Print cluster api raw json data",
+				Flags: append([]cli.Flag{
+					&cli.StringFlag{
+						Name:        "creds",
+						Usage:       "Path to GCP JSON credentials file",
+						Destination: &config.CredentialsFile,
+					},
+					&cli.StringFlag{
+						Name:        "project",
+						Aliases:     []string{"p"},
+						Usage:       "Name of a GCP project",
+						Destination: &config.ProjectName,
+					},
+					&cli.StringFlag{
+						Name:        "name",
+						Aliases:     []string{"n"},
+						Usage:       "Name of a GKE cluster to review",
+						Destination: &config.ClusterName,
+					},
+					&cli.StringFlag{
+						Name:        "location",
+						Aliases:     []string{"l"},
+						Usage:       "GKE cluster location (region or zone)",
+						Destination: &config.ClusterLocation,
+					},
+				}, getPolicySourceFlags(config)...),
+				Action: func(c *cli.Context) error {
+					defer p.Close()
+					if err := p.LoadCliConfig(config); err != nil {
```

Comment:
Pls merge main branch as `LoadCliConfig` func now takes additional parameter to pass validation function.
You can pass `nil` but I suggest to create one i.e. `ValidateClusterJSONDataConfig`